File: Leetcode/1295/Linear_Search_Algo/main.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
    let count = 0;
    for (let num of nums) {
        // Handle the case where num is less than 1
        let digitCount = num < 1 ? 1 : Math.trunc(Math.log10(num) + 1);
        if (digitCount % 2 === 0) {
            count++;
        }
    }
    return count;
};

File: Leetcode/1295/Linear_Search_Algo/main.go
func findNumbers(nums []int) int {
    var count int = 0

    for _,num := range nums {
        numm := int(math.Log10(math.Abs(float64(num))) + 1)
        if numm%2==0 {
            count++
        }
    }
    return count   
}